Fix spellings and grammar checks (#110)
kbrew-dev/kbrew: README.md

![kbrew-logo](./images/kbrew-logo.png)

> kbrew is a Kubernetes tool that aims to make installing any complex stack in any cloud possible with `one step` (And yes we are definitely inspired by Homebrew from MacOS)

Reviewer: ```suggestion
kbrew is a Kubernetes tool that aims to make installation of any complex stack in any cloud a `one step` task/process (And yes we are definitely inspired by Homebrew from MacOS)
```
